### ISSUE TITLE
hi is the resnet part for extract features use pretrained model?(im too noob to make sure that i understood the code right..) )

### ISSUE THREAD

**Issue by lvcat** [yuxiangsun/RTFNet#3]

**Comment by yuxiangsun** [comment]
Yes, it loads the pre-trained ResNet weight using: `pretrained=True`

**Comment by lvcat** [comment]
> Yes, it loads the pre-trained ResNet weight using: `pretrained=True`

thanks, it matters to me.  : )

**Comment by lvcat** [comment]
> Yes, it loads the pre-trained ResNet weight using: `pretrained=True`

hi me again. Would u mind provide the training log that record the loss and acc. i want to reference it for my training= = .  Thank u very much.QAQ

**Comment by lvcat** [comment]
您好 我按照您论文里的参数重新训练试了一下，跟您论文里的以及给出的权重有一些差距，有什么我应该注意的小细节吗

**Comment by yuxiangsun** [comment]
You may want to set `drop_last` as `false` for the train dataloader. 
But I think that the initialization scheme mainly introduces the variance, you may try training it again.
